swagger.yaml

```diff
@@ -5717,6 +5717,24 @@ components:
           format: uuid
           readOnly: true
           description: Id of ExhibitionDeviceGroup that ExhibitionDevice belongs to
+          
+    MqttDeviceStatus:
+      type: object
+      description: |
+        MQTT notification message about devices status. This is actually not part of the REST API spec but a realtime notification via MQTT channel.
+        It is periodically sent by all devices to the server.
+      properties:
```

Comment:
I would imagine that deviceId, status and version would all be required

Reply:
Resolved.